java/src/test/java/vowpalWabbit/learner/VWLearnersTest.java

```diff
@@ -32,7 +32,7 @@
     @Test
     public void testWrongType() {
         thrown.expect(ClassCastException.class);
-        VWScalarsLearner learner = VWLearners.create("--cb 4 --quiet");
+        VWScalarsLearner learner = VWLearners.create("--cb 4 --quiet --force_legacy");
```

Comment:
Rename flag to "--cb-force-legacy" since it is cb specific.

Reply:
done